donner.go

```diff
@@ -65,17 +66,17 @@ func main() {
 }
 
 // execCommand dispatches the call to the OS
-func execCommand(cfg *Cfg, command []string, strict bool) error {
-	if len(command) < 1 {
+func execCommand(cfg *Cfg, cliArgs []string, strict bool) error {
+	if len(cliArgs) < 1 {
```

Comment:
I find `cliArgs` a bit confusing here. What is passed to the function is essentially the command we want to wrap and execute